backend/src/apis/contacts/helpers/findContact.ts

```diff
@@ -0,0 +1,7 @@
+import Contact from '../../../lib/models/contact';
+import { IContact } from '../../../lib/interfaces/IContact';
+
+//function receives object with query attributes...eg. {email:"abc@gmail.com"}
```

Comment:
Not necessary for all your methods. Where applicable, I would suggest using JSDoc format. vscode would also understand it.